packages/storybook8/stories/Components/VideoGallery/Docs.mdx

```diff
@@ -154,51 +160,68 @@ The vertical gallery can be most useful when the application is running a very w
       src="images/short-screen-share.png"
       alt="Fill to frame in VideoGallery"
     />
-    <>❌ Wide screen share appearence horiztonal gallery</>
+    <>
+      <br />❌ Wide screen share appearence horiztonal gallery
+    </>
   </Stack>
   <Stack horizontalAlign="center">
     <Image
       style={{ width: '100%', height: 'auto' }}
       src="images/short-screen-share-vertical.png"
       alt="Fill frame menu item in VideoGallery"
     />
-    <>✅ Wide screen share appearence vertical gallery</>
+    <>
+      <br />✅ Wide screen share appearence vertical gallery
+    </>
   </Stack>
 </Stack>
 
-## Screen Sharing Experience
+## Content Sharing Experience
 
 The screen shared is the only element placed in the GridLayout and all remote participants are placed in the horizontal gallery in the lower section. To be able to view this screen share, the sharing participant should have their `isScreenSharingOn` prop set to true as well as a defined `screenShareStream` prop (see `localParticipant` and `remoteParticipants` props).
 
+- <strong>Screen Sharing:</strong> Share your local screen with participants in real-time or view remote screen shares.
+  For more details, see the sections below.
+- <strong>PowerPoint Live:</strong> Exclusively view remote PowerPoint presentations.
+  <a href="?path=/docs/concepts-powerpoint-live--docs">Learn more about PowerPoint Live</a>.
+
+### Supported Features
+
+Currently, the VideoGallery component supports two primary features to enhance collaboration and presentation quality: screen sharing and PowerPoint Live.
+
```

Comment:
The two bullet points should be here.

Reply:
i must have missed this change. Should be in now. I recall making an unordered list before but it might not have been commited.